### PR TITLE
draft Statements page + trace docs

### LINKED ISSUE #4736: Trace Information on Statements Page in Admin UI
Background: https://airtable.com/tblD3oZPLJgGhCmch/viw1DKmbKhg2MIECH/recDDX3AlQ7rNqPja

Description: On the statements page in the Admin UI, users will be able to view a trace of statement(s). The traces displayed will either be a randomly sample to help users understand the tool or specifically requested ones.

Addressing the problem of observability, the traces will help users understand why queries are slow, whether your mental model of CRDB is correct (e.g. was this query served from the locality that I want).

Internally, the traces will help CRDB engineers find ways to improve CRDB's query execution.

### LINKED ISSUE #6941: ui: Statements tracing
PR: https://github.com/cockroachdb/cockroach/pull/45799

From release notes:
> You can now access the Activate Diagnostics modal from the [Statements page](../v20.1/admin-ui-statements-page.html), through the **Activate** link. [#45799][#45799] {% comment %}doc{% endcomment %}

### PR DESCRIPTION
**Pull request by taroface**
Fixes #4736.
Fixes #6941.

### CODE REVIEW

File: v20.1/admin-ui-statements-page.md [hunk 1]
```diff
@@ -7,135 +7,154 @@ toc: true
...
-Latency | The average service latency of the SQL statement (or multiple statements having the same fingerprint) within the last hour or the [specified time interval](#limitation). <br><br>The latency is represented in two ways: The numerical value shows the mean latency, while the horizontal bar is color-coded (blue indicates the mean value and yellow indicates one standard deviation of the mean value). The bar also helps you compare the mean latencies across all SQL fingerprints in the table. <br><br>You can sort the table by latency.
+The **Diagnostics** section of the Statement Details page allows you to activate and view diagnostics for the SQL statement fingerprint.
 
-### Execution Count
+When you activate diagnostics for a fingerprint, CockroachDB looks for SQL queries that match this fingerprint. On the next match, information about the SQL statement is written to a diagnostics bundle that you can download. This bundle consists of a JSON file that contains a distributed trace of the SQL statement.
```
Comment (dhartunian) [inline v20.1/admin-ui-statements-page.md]: is it worth clarifying here that the diagnostics will be collected a maximum of N times for a given activated fingerprint where N is the number of nodes in your cluster?

**Comment by taroface** [comment]
Thanks for the awesome review @dhartunian @piyush-singh! Still had some follow-up questions on these.

**Comment by taroface** [comment]
@piyush-singh I have updated with the new bundle info whenever you want to do a (hopefully) final review!

File: v20.1/admin-ui-statements-page.md [hunk 2]
```diff
@@ -7,135 +7,172 @@ toc: true
...
+Statement | SQL statement [fingerprint](#sql-statement-fingerprints).<br><br>To view additional details of a SQL statement fingerprint, click this to open the [**Statement Details** page](#statement-details-page).
+Txn Type | Type of transaction (implicit or explicit). Explicit transactions refer to statements that are wrapped by [`BEGIN`](begin-transaction.html) and [`COMMIT`](commit-transaction.html) statements by the client. Explicit transactions employ [transactional pipelining](architecture/transaction-layer.html#transaction-pipelining) and therefore report latencies that do not account for replication.<br><br>For statements not in explicit transactions, CockroachDB wraps each statement in individual implicit transactions. 
+Retries | Cumulative number of [retries](transactions.html#transaction-retries) of statements with this fingerprint within the last hour or specified [time interval](#time-interval).
```
Comment (rmloveland) [inline v20.1/admin-ui-statements-page.md:73]: These and other uses of "the last hour or specified time interval" could be changed to say only "the specified time interval [LINK]" so if/when the setting's default value changes, docs do not need to be updated.  Judgment call

File: v20.1/query-behavior-troubleshooting.md [hunk 3]
```diff
@@ -20,17 +20,19 @@ To detect whether your cluster has slow queries, check the [service latency grap
...
+You can then use the [Statement Details](admin-ui-statements-page.html#statement-details-page) page to drill down to individual statements and [activate diagnostics](admin-ui-statements-page.html#diagnostics) for the statement that you can send to our [support team](support-resources.html) for analysis.
 
-If you need help interpreting the output of the `EXPLAIN ANALYZE` statement, [contact us](file-an-issue.html).
+You can also use an [`EXPLAIN ANALYZE`](explain-analyze.html) statement, which executes a SQL query and returns a physical query plan with execution statistics. Query plans provide information around SQL execution, which can be used to troubleshoot slow queries by figuring out where time is being spent, how long a processor (i.e., a component that takes streams of input rows and processes them according to a specification) is not doing work, etc.
+
+If you need help interpreting the output of the `EXPLAIN ANALYZE` statement, [contact us](support-resources.html).
```
Comment (rmloveland) [inline v20.1/query-behavior-troubleshooting.md:27]: Suggest changing "contact us" to "contact our support team" so it matches the phrasing in the previous paragraph; plus it makes clearer who / what exactly you are contacting

File: v20.1/query-behavior-troubleshooting.md [hunk 4]
```diff
@@ -20,17 +20,19 @@ To detect whether your cluster has slow queries, check the [service latency grap
...
 
 ## Query is sometimes slow
 
 If the query performance is irregular:
 
 1.  Run [`SHOW TRACE`](show-trace.html) for the query twice: once when the query is performing as expected and once when the query is slow.
 
-2.  [Contact us](file-an-issue.html) to analyze the outputs of the `SHOW TRACE` command.
+2.  [Contact us](support-resources.html) to analyze the outputs of the `SHOW TRACE` command.
```
Comment (rmloveland) [inline v20.1/query-behavior-troubleshooting.md:35]: Same re: "contact us" -> "contact our support team"

Conversely, I guess everything could be changed to say "contact us".  Your call ofc!

**Pull request merged**